EXAMPLES.md

```diff
@@ -0,0 +1,74 @@
+# Examples
+
+- [Use a custom service name](#use-a-custom-service-name)
+- [Include additional attributes](#include-additional-attributes)
+- [Share items with other apps and extensions using an access group](#share-items-with-other-apps-and-extensions-using-an-access-group)
+- [Share items with other devices through iCloud synchronization](#share-items-with-other-devices-through-icloud-synchronization)
+- [Restrict item accessibility based on device state](#restrict-item-accessibility-based-on-device-state)
+- [Require Touch ID / Face ID to retrieve an item](#require-touch-id--face-id-to-retrieve-an-item)
+
+---
+
+### Use a custom service name
```

Comment:
We have an h3 following an h1 without an h2, should we add a h2 or change all h3's to h2's ?

Reply:
Good catch! Fixed in https://github.com/auth0/SimpleKeychain/pull/175/commits/e5505d895860ee03f48631603dc000666bd3bd98